Update ChessBoard.cs

mauer01 opened the pull request:
i am pretty sure this should do it

mauer01 committed:
Update ChessBoard.cs

i am pretty sure this should do it

penteract commented on FiveDChessDataInterface/Types/ChessBoard.cs:
```diff
@@ -49,7 +49,7 @@ public ChessBoard(ChessBoardMemory mem, int width, int height)
                             x += 1;
                         }
-                        pieces += $"{i}";
+                        pieces += $"{i}"+p.FENSymbol();
```
`p` is still the the same as when p.isEmpty was true, and this doesn't consider the case where `x==width` and there isn't another piece on this row to include. One solution would be:
```
pieces += $"{i}";
if(x<width) pieces += Pieces[x*height+y].FENSymbol();
```

penteract committed:
Fix fix for FEN

Apply correct fix for the bug where FEN doesn't show pieces with an empty space to their left.
